### DISCUSSION TITLE
Disable upload option in files field

### DISCUSSION THREAD

**Discussion by Dumpstar56** [directus/directus#14950] [Feature Requests]
I think it would be great to be able to force users to select from existing files in the file directory for file and files fields. In our project, file uploads are managed centrally and all necessary files are uploaded for users. So all they need is already in the file directory. To prevent users from uploading unnecessary files, I would love to be able to disable the upload function in file and files relation fields, probably in the Interface options.

**Comment by br41nslug** [comment]
You can block users from uploading files by taking away the create/delete/update permissions of `directus_files` for that role

**Reply by Dumpstar56** [reply]
Yeah, that works. Thank you for that info! Not even seeing the option of uploading for single files would be great still, but this absolutely does the trick.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
